### DISCUSSION TITLE
排除人工，有没有办法把uniapp转成taro实现？

### DISCUSSION THREAD

**Discussion by ignite-404** [NervJS/taro#11400] [Q & A]
排除人工，有没有办法把uniapp转成taro实现？

**Comment by yueim** [comment]
几乎没有可能。
uni 下可能引入各种第三方插件。
而好多第三方插件是 uni only 的。
所以自动转换，几乎是不可能的。
看项目实际情况，人肉转换也许可行。

**Comment by tourze** [comment]
人肉转换，样式问题都是个大问题。。。
